backend/src/disk/fsck.rs

```diff
@@ -0,0 +1,78 @@
+use std::ffi::OsStr;
+use std::path::Path;
+
+use color_eyre::eyre::eyre;
+use tokio::process::Command;
+use tracing::instrument;
+
+use crate::{Error, ResultExt};
+
+#[derive(Debug, Clone, Copy)]
+pub enum RequiresReboot {
```

Comment:
Comment: Just weird that we are doing this just for the or operator
Side: Could rethink the bool as 


```rs
pub enum RebootAction;
/// Then return Option<RebootAction>?

/// Or
pub enum Action {
  Nothing,
  Reboot,
}
```
Doesn't solve the boolean issues, except the blindness.
Since this is not far from a boolean, and we just get a smarter wrapper
```rs
pub struct RequiresReboot(bool);
````